Specs/Scene/BillboardCollectionSpec.js

```diff
@@ -2491,6 +2491,21 @@ describe(
         expect(b._clampedPosition).toBeUndefined();
       });
 
+      it("disableDepthTest after another function", function () {
+        const b = billboardsWithHeight.add({
+          heightReference: HeightReference.CLAMP_TO_GROUND,
+          position: Cartesian3.fromDegrees(-122, 46.0),
+          disableDepthTestDistance: Number.POSITIVE_INFINITY,
+        });
+        expect(scene.globe.callback).toBeDefined();
```

Comment:
Out of context, I would have suggested keeping the test limited to the billboard collection itself to avoid coupling this test to behavior with another class. However, I see this is the same way we are testing `"callback updates the position"`, so I think its fine to keep consistant.

A better way in both cases might be to call `renderForSpecs` after setting the `heightReference` and confirming that `._clampedPosition` has or hasn't changed based on the expected behavior, but there may be reason why `"callback updates the position"` wasn't written that way to begin with that I'm not aware of.

Reply:
That makes sense; I've updated the test changes, though only in the test I made and not in "callback updates the position."